src/systems/filecoin_mining/storage_mining/storage_mining_subsystem.go

```diff
@@ -125,3 +128,120 @@ func (sms *StorageMiningSubsystem_I) PrepareNewTicket(randomness util.Randomness
 
 	return newTicket
 }
+
+// TODO Import IPLDStore
+var localIPLDStore ipld.GraphStore
+
```

Comment:
@jzimmerman Check out the below if you can!

Reply:
If this is the right kind of pattern, then the ipld store should be added as a member of StorageMiningSubsystem and then this would look pretty straightforward.

Can we get some kind of `Get(cid, interface{})` that decodes into the second parameter?

Reply:
I actually think the IPLD store might be added to the node or some subsystem available to all others so they can all access the single one...

Looking into the Get you mention, it's a good idea. I just worry that .ids don't support generics so passing in the interface type may not work cc @jzimmerman

Reply:
You can definitely pass interface types, even though we don't yet have support for generics -- just use a named Go type such as `util.Any`. However, it's the method implementation itself that concerns me more, since Go doesn't support generics either (and reflection in cases like this seems quite error-prone).

For the moment, I would suggest having `ipldStore.Get(CID)` return a `util.Serialization`, which can then immediately be deserialized via the appropriate, auto-generated type-specific methods (e.g. `Deserialize_StorageMinerActorState` in the code below).